### DISCUSSION TITLE
Filter two dates columns

### DISCUSSION THREAD

**Discussion by moayadsquad** [directus/directus#12808] [Archive]
How in directus to filter two dates column by range values?

my query 
filter[start_date][_gte] = 2022-04-01
filter[end_date][_lte] = 2022-04-19

I does not work for two columns fields, any other query to get my result back?

**Comment by rijkvanzanten** [comment]
What doesn't work? Do you not get the results you're looking for? What types are those fields? What are the names of your fields?
